modules/UI/videolayout/VideoLayout.js

```diff
@@ -176,9 +176,7 @@ var VideoLayout = {
         let localId = APP.conference.getMyUserId();
         this.onVideoTypeChanged(localId, stream.videoType);
 
-        if (!stream.isMuted()) {
-            localVideoThumbnail.changeVideo(stream);
-        }
+        localVideoThumbnail.changeVideo(stream);
```

Comment:
The change was added specifically for the case of changing video devices while in audio only mode. The user enters audio only mode and opens device selection, chooses a new video device, and confirms the change. At that point a new track is created and muted and set as the current track. With the isMuted check left in place, the local video container would hold on to the previous local track so leaving audio only mode would result in a black/empty local video, which might also be the video on large display. Without the isMuted check, the local video container updates its reference. No adverse effects were noticed with the line removed, which was added initially for starting a conference video muted.